src/repair_schedule.jl

```diff
@@ -0,0 +1,54 @@
+include("../src/NursesScheduling.jl")
+using .NurseSchedules
+
+import Base.in
+
+Shifts = Array{String,2}
+BestResult = @NamedTuple{shifts::Shifts, score::Int}
+
+function repair_schedule(schedule_data)
+    ITERATION_NUMBER = 50
+    TABU_MAX_SIZE = 10
```

Comment:
stałe gdzieś indziej

Reply:
no pewnie jako argumenty do tych funkcji by się przydało zrobić, ale to zostawiłem na później